custom_components/waste_collection_schedule/waste_collection_schedule/source/lewisham_gov_uk.py

```diff
@@ -0,0 +1,111 @@
+import datetime
+import logging
+import re
+
+import requests
+from waste_collection_schedule import Collection
+
+TITLE = "London Borough of Lewisham"
+DESCRIPTION = (
+    "Source for services from the London Borough of Lewisham"
+)
+
+URL = "lewisham.gov.uk"
+TEST_CASES = {
+    "houseNumber": {"post_code": "SE41LR", "number": 4},
+    "houseName": {"post_code": "SE233TE", "name": "The Haven"},
+    "houseUprn": {"uprn": "10070495030"}
+}
+
+API_URLS = {
+    "address_search": "https://lewisham.gov.uk/api/AddressFinder?postcodeOrStreet={postcode}",
```

Comment:
Please pass URL parameters using the params argument instead of string concatenation: https://requests.readthedocs.io/en/latest/user/quickstart/#passing-parameters-in-urls

Same is true for the following requests.

Reply:
Done. Hope this is ok.